packages/core/test/synchronizerAttesters.test.ts

```diff
@@ -329,9 +334,7 @@ describe('Synchronizer watch multiple attesters', function () {
             const attesterId = BigInt(accounts[i].address).toString()
             const { publicSignals, proof, epochKey, epoch } =
                 await userState.genEpochKeyProof({ attesterId })
-            await unirepContract
-                .verifyEpochKeyProof(publicSignals, proof)
-                .then((t) => t.wait())
+            unirepContract._updateEpochIfNeeded(attesterId)
```

Comment:
I don't think this line is needed. The `attest` function will call this. It's also not `await`ed

Reply:
Deleted `unirepContract._updateEpochIfNeeded(attesterId)`